src/action-controller/error_handler.cr

```diff
@@ -46,17 +57,7 @@ class ActionController::ErrorHandlerDevelopment
     response = context.response
     reset(response)
 
-    if accepts_formats(context.request).includes?("application/json")
-      response.content_type = "application/json"
-      {
-        error:     ex.message,
-        backtrace: ex.backtrace?,
-      }.to_json(response)
-    else
-      response.content_type = "text/plain"
-      response.print("ERROR: ")
-      ex.inspect_with_backtrace(response)
-    end
```

Comment:
Will there still be a way to receive the stack trace as JSON in development?

Reply:
I can add that back but then this change has no benefits for our purposes as we're entirely API driven
I tested it and it renders pretty well in the preview pane but is also pretty limited there due to no JS support

Reply:
might be worth not having JSON errors in development with this change
https://github.com/crystal-loot/exception_page/pull/33

individual apps can also add their own exception handlers, this is just the default